eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -23,6 +24,10 @@
 use eZ\Publish\API\Repository\Values\Content\Trash\SearchResult;
 use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
 use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\PermissionCriterionResolver;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalAnd as CriterionLogicalAnd;
```

Comment:
```suggestion
use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
```
in many places in the kernel it is used like this